src/tools/rust-analyzer/crates/proc-macro-srv/proc-macro-test/imp/src/lib.rs

```diff
@@ -1,6 +1,10 @@
 //! Exports a few trivial procedural macros for testing.
 
+#![cfg(feature = "sysroot-abi")]
```

Comment:
```suggestion
#![allow(unexpected_cfgs)]
#![cfg(feature = "sysroot-abi")]
```
That is this might just work?